src/mod/endpoints/mod_sofia/mod_sofia.h

```diff
@@ -192,6 +192,19 @@ struct sofia_private {
 	int is_static;
 	switch_time_t ping_sent;
 	char *rfc7989_uuid;
+	int rtp_timeout_sec;
```

Comment:
Use different variable name for rtp media timeout to avoid confusion. Take note that rtp media timeout uses msec